fwl-expect package (#1)
fewlines-education/camp2-helpers: fwl-expect/README.md

> # Fewlines Expect

Reviewer: I would suggest Camp2 Helpers since there will be only one repo for all packages

This package modifies the [expect](https://jestjs.io/docs/en/expect) and wrap it in a custom package in order to give an extra option to add a message when jest tests fails in Typescript.

> It modifies the package [jest-expect-message](https://www.npmjs.com/package/jest-expect-message) for custom use.

Reviewer: ```suggestion
It is based on the package [jest-expect-message](https://www.npmjs.com/package/jest-expect-message).
```

## Usage

`fwl-expect` is not yet available on npm.

(the rest is commented and will be used once the package has a more final form)

> <!-- To install it, add this in your `package.json` file:

Reviewer 2: We should remove all of that 🙂 ⬇️
